internal/store/spicedb/relation_repository.go

```diff
@@ -102,15 +107,21 @@ func (r RelationRepository) AddV2(ctx context.Context, rel relation.RelationV2)
 }
 
 func (r RelationRepository) Check(ctx context.Context, rel relation.Relation, act action.Action) (bool, error) {
+	consistency := &authzedpb.Consistency{}
+	if r.fullyConsistent {
```

Comment:
Do we need if condition here as default bool value will be false.
Is there any performance hit event if we are passing false?

Reply:
Good idea, refactored it as `r.getConsistency()` as I would still want to fallback to default behaviour

```go
func (r RelationRepository) getConsistency() *authzedpb.Consistency {
	if !r.fullyConsistent {
		return &authzedpb.Consistency{}
	}
	return &authzedpb.Consistency{Requirement: &authzedpb.Consistency_FullyConsistent{FullyConsistent: true}}
}
```